apps/admin/app/students/_components/StudyProvider.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
+
+import { atom, createStore, Provider } from "jotai";
+import type { ReactNode } from "react";
+
+const studyIdStore = createStore();
+
+export type StudyAtomprops = {
+  studyId: number;
+  title: ReactNode;
+};
+
+export const studyAtom = atom<StudyAtomprops>();
+studyIdStore.set(studyAtom, undefined);
+
+export const StudyProvider = ({ children }: { children: ReactNode }) => {
```

Comment:
p5;
요렇게 적용할 수도 있을 거 같아요!
```suggestion
export const StudyProvider = ({ children }: PropsWithChildren}) => {
```

Reply:
반영했습니당 [b2f86bd](https://github.com/GDSC-Hongik/wow-class/pull/140/commits/b2f86bd1efcb9233db186d66ef4e7f0c66798878)